web/assets/js/app/Browser/component/controller/NodeController.js

```diff
@@ -3,16 +3,18 @@ define([
 ], function(angular) {
     'use strict';
 
-    function NodeController($scope, $state, $graph, $search, $fuzzyFilter, $notification, $treeFactory, $timeout, $q) {
+    function NodeController($scope, $state, $graph, $search, $fuzzyFilter, $treeFactory, $timeout, $q, $success, $error, $errorFromResponse) {
```

Comment:
To have more dedicated services, this way we can decide to improve `$error` with more options for example in the future. I agree it can be sometimes harder to understand, but it is not used that much et the name are explicit.